doc/user/ospfd.rst

```diff
@@ -699,6 +699,15 @@ Interfaces
    OSPF (:ref:`redistribute-routes-to-ospf`). This is the only way to
    advertise non-OSPF links into stub areas.
 
+.. clicmd:: ip ospf prefix-suppression [A.B.C.D]
+
+   Configure OSPF to not xadvertise the IPv4 prefix associated with the
```

Comment:
xadvertise?

Reply:
Fixed - this was a late edit that wasn't carried out correctly.